source/tutorial/kerberos.txt

```diff
@@ -0,0 +1,351 @@
+                   port: 3306
+                 security:
+                   enabled: true
+                   defaultSource: "$external"
+                   defaultMechanism: "GSSAPI"
+                   gssapi:
+                     hostname: "BI.EXAMPLE.COM"
+                     serviceName: "mongosql"
+                 mongodb:
+                   net:
+                     uri: "mongodb://BI.EXAMPLE.COM/"
+                     auth:
+                       username: "grace.smith@EXAMPLE.COM"
+                       password: "abc123!"
+                       source: "$external"
+                       mechanism: "GSSAPI"
+                 processManagement:
+                   service:
+                     displayName: "MongoDB BI Connector"
+
+            - Open :guilabel:`Services`.
+            - Set MongoDB to 
+              :guilabel:`Log on as: This account:` 
+              ``mongodb@EXAMPLE.COM`` 
+            - Set |bi| to 
+              :guilabel:`Log on as: This account:` 
+              ``mongosql@EXAMPLE.COM``
+            - Start the MongoDB and |bi| services.
+
+         .. seealso::
+
+            To learn how to configure Active Directory to 
+            manage your MongoDB instance, see
+            :manual:`Configure MongoDB with Kerberos Authentication and Active Directory Authorization 
+            </tutorial/kerberos-auth-activedirectory-authz/>`.
+
+     - id: macos
+       content: |
+
+         MongoDB Configuration
+         ~~~~~~~~~~~~~~~~~~~~~
+
+         - Make sure your :binary:`~bin.mongod` and
+           :binary:`~bin.mongos` instances are running
+           `MongoDB Enterprise <http://www.mongodb.com/products/mongodb-enterprise?jmp=docs>`__.
+         - Configure your :binary:`~bin.mongod` and
+           :binary:`~bin.mongos` instances to use
+           :manual:`Kerberos (GSSAPI) authentication </core/kerberos>`.
+         - Verify you set the following options the MongoDB
+           :manual:`configuration file </reference/configuration-options>`:
+
+           .. code-block:: yaml
+
+              net:
+                bindIp: <fullHostnameOfBIConnector>
+              setParameter:
+                authenticationMechanisms: "GSSAPI"
+
+         - `Create a keytab file`_ on the |kdc| that has the 
+           needed Kerberos :ref:`service principals <kerberos-service-principal>`
+           for ``mongosql`` and ``mongosql2``.
+
+           .. example::
+
+              .. code-block:: sh
+
+                 mongosql/<fullHostnameOfBIConnector>@<KERBEROS_REALM>
+
+         BI Connector Configuration
+         ~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+         - Copy the ``keytab`` file you created for your MongoDB
+           deployment to the same host that serves the |bi-short|.
+         - If your keytab does not use the default name 
+           (``krb5.keytab``), you must set the
+           `KRB5_KTNAME <https://web.mit.edu/kerberos/krb5-latest/doc/basic/keytab_def.html>`_ system variable.
```

Comment:
can we use `environment variable` instead of `system variable` here and elsewhere? I think it will help prevent users confusing this with MySQL's system variables